theano/gpuarray/tests/test_linalg.py

```diff
@@ -112,3 +112,83 @@ def test_linalgerr_solve(self):
 
         fn = theano.function([A, b], [solver], mode=mode_with_gpu)
         self.assertRaises(LinAlgError, fn, A_val, x_val)
+
+
+class TestGpuCholesky(unittest.TestCase):
+
+    def setUp(self):
+        utt.seed_rng()
+
+    def get_gpu_cholesky_func(self, lower=True, inplace=False):
+        # Helper function to compile function from GPU Cholesky op. 
```

Comment:
There is a trailing white space at the end of the comment, that makes flake8 fail. Make sure there are no trailing spaces at the end of comments you have added (lines 123, 130, 140, 146, 153, 160, 168, 188 and 194, at least!).